api/tests/unit/features/multivariate/test_unit_multivariate_views.py

```diff
@@ -10,13 +17,50 @@ def test_multivariate_feature_options_view_set_get_permissions():
     permissions = view_set.get_permissions()
 
     # Then
-    assert len(permissions) == 2
-    assert isinstance(permissions[1], NestedProjectPermissions)
-    assert permissions[1].action_permission_map == {
+    assert len(permissions) == 1
+    assert isinstance(permissions[0].op1, NestedProjectPermissions)
```

Comment:
What is the `op1` attribute?

Reply:
It stands for operator 1 I think https://github.com/encode/django-rest-framework/blob/master/rest_framework/permissions.py#L41

Reply:
What is it though? I'm assuming it's the first permission class in the `Class1 | Class2` logic of the permission classes? Can we add a comment explaining as it's a bit cryptic at the moment?

Reply:
Yep, good point. I have added a note